### PR TITLE
Feature/itm 304

### PR DESCRIPTION
**Pull request by mrb-itminds**
depends on #52 
Branch went a bit out of scope since I also did a lot of work on the flow of creating and application, then creating af AppToken for it, and selecting a apptoken from a list and then adding AppTokenActions via the ServiceLibrary(modal).

The Pending apptokens list is currently its own Page with a simple list of apptokens showing apptokens that has actions that needs reviewing. Each listitem has a button to the right thats gonna open the Approve/Decline Modal (feature 305)

### CODE REVIEW

File: backend/Application/AppTokens/Queries/GetAppTokens/GetAppTokensQuery.cs [hunk 1]
```diff
@@ -32,7 +37,27 @@ public async Task<List<AppTokenIdDto>> Handle(GetAppTokensQuery request, Cancell
...
+          }
+
+          if (isPending)
+          {
+            pendingTokens.Add(token);
+          }
+        }
+
+        return pendingTokens;
+
```
Comment (objmj-itminds) [inline backend/Application/AppTokens/Queries/GetAppTokens/GetAppTokensQuery.cs:60]: I believe you can get this using LINQ. Try and then we can look at it together.

Comment (mrb-itminds) [reply backend/Application/AppTokens/Queries/GetAppTokens/GetAppTokensQuery.cs]: Worked like a charm with LINQ

File: frontend/src/components/ApplicationTable/ApplicationTableMenus/ApplicationItemMenus/AddAppTokenTriggerBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      });
+    } catch (error) {
+      toast({
+        description: `CreateAppToken responded: ${error}`,
+        status: "error",
+        duration: 5000,
+        isClosable: true
+      });
+    }
+    fetchAppTokens;
```
[the remark this answers is not in the document]
Comment (mrb-itminds) [reply frontend/src/components/ApplicationTable/ApplicationTableMenus/ApplicationItemMenus/AddAppTokenTriggerBtn.tsx:49]: Yea forgot, noticed it on a later branch aswell xD

File: backend/Web/Controllers/ApplicationController.cs [hunk 3]
```diff
@@ -51,9 +51,10 @@ public async Task<ActionResult<int>> CreateAppTokenActions([FromRoute] int token
       return await Mediator.Send(command);
     }
     [HttpGet("AppTokens")]
-    public async Task<ActionResult<List<AppTokenIdDto>>> GetAllAppTokens()
+    public async Task<ActionResult<List<AppTokenIdDto>>> GetAllAppTokens(bool onlyPending = false)
```
[the remark this answers is not in the document]
Comment (mrb-itminds) [reply backend/Web/Controllers/ApplicationController.cs:54]: Makes sense

File: frontend/src/components/PendingApprovals/PendingListItem.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        <Flex align="center" m="4" p="2" borderWidth="1px" borderRadius="sm">
+          <Flex flexDirection="column">
+            <Box>{`Token: ${appToken.id} with ${appToken.appTokenActions.length} actions`}</Box>
+            <Box padding="2">{appToken.description}</Box>
+          </Flex>
+          <Spacer />
+          <Button>Review</Button>
+        </Flex>
+      ) : (
+        <></>
```
[the remark this answers is not in the document]
Comment (mrb-itminds) [reply frontend/src/components/PendingApprovals/PendingListItem.tsx:22]: I see, a lot prettier too

**Review by objmj-itminds — APPROVED** [review]

**Pull request merged**